services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -84,20 +85,16 @@ def create_as_initially_ok(cls) -> "Status":
         return initial_state
 
 
-class DockerStatus(str, Enum):
-    CREATED = "created"
-    DEAD = "dead"
-    EXITED = "exited"
-    PAUSED = "paused"
-    REMOVING = "removing"
-    RESTARTING = "restarting"
-    RUNNING = "running"
+DockerStatus = Status2
 
 
 class DockerContainerInspect(BaseModel):
     status: DockerStatus = Field(
         ...,
-        scription="status of the underlying container",
+        description="status of the underlying container",
```

Comment:
Q: why not using docker_rest_api.py ContainerState directly?

Reply:
I'm actually using it now